include/client.h

```diff
@@ -646,35 +651,37 @@ struct Client {
                             || HasPriv(sptr, PRIV_SEE_OPERS)))
 
 /** Clear the client's net.burst in-progress flag. */
-#define ClearBurst(x)           ClrFlag(x, FLAG_BURST)
+#define ClearBurst(x)            ClrFlag(x, FLAG_BURST)
```

Comment:
Could you revise this to not touch whitespace on lines that don't have text changes?